server/GameServer.cpp

```diff
@@ -81,7 +81,7 @@ void GameServer::Pump()
 		}
 
 		// Do periodic updates
-		DoTimedUpdates();
+		CheckTimedUpdates();

```

Comment:
I'm not overly fond of this rename. The word "check" implies that it doesn't actually do anything. This method does have rather involved side effects though.

I see there are later updates to split the function. If's fine for a "Do" method to call a "Check" method, though it's probably not good practice for a "Check" method to call a "Do" method.